packages/components/src/toggle-control/README.md

```diff
@@ -22,6 +25,38 @@ const MyToggleControl = withState( {
 ) );
 ```
 
+To use this inside a block to toggle a saveable value, use block attributes, e.g.
```

Comment:
While I think this is valuable, I'm not sure it's the right place for it. Why just the toggle control, why not all *controls?. I think the components documentation should probably stay agnostic to the context where the component is used.

Reply:
1. People don't know much about gutenberg so they look up the individual components for how to use them in their blocks
2. Examples are seen as canonical. You and I might understand that `withState` is a tool being used here, but the general consensus to those not in the know is that it's a requirement for using toggle control components.
3. I have a busy working day, doing this for every component is a nice idea, it also sounds like a full day of work, I've a deadline tomorrow so my project manager might want to have words :)

If we want to make the component documentation agnostic then removing the `withState` and having placeholder variables such as:

```js
const setToggleValue = ...
const currentValue = ...;
```

Otherwise the context agnostic approach is not being followed here, and, is actively misleading. I understand there's a certain appeal to keeping components in their pristine context-less state, but people need examples. Context in and of itself can be useful.

If the case is that all documentation should be consistent, then I agree, but that shouldn't prevent fixes to technical documentation. "Either all components can have it or none of them can have it" basically means none of them will have it.

Reply:
And if this isn't the place, then where is? And should all these components not link there?

This isn't the first time I've seen components documented in isolation causing issues. Important details are left out because they're not directly a part of the component, but rather their dependencies.

Reply:
It's not about writing less documentation, it's because it doesn't feel like the right place for it for me at least. Maybe it's about documentation organization.

For me you want to write a block:

 - Learn React that way you learn how components work (so we potentially need links to React docs) and it's not necessary to repeat the same documentation over and over for each component.
 - Learn the Block API and learn that you can use any React component there including the provided ones. Link to the components docs there with potentially an example of how to use one of these.

So I see several things:

 - Do we have links to React docs anywhere? 
 - Do we have links to components from the "create blocks tutorials"?

To put it differently (and also a bit controversially :)) Do we explain how to use PHP Classes and PHP function in each WordPress PHP API? 

And potentially to get the best of both world, link to React docs at the top of each component README?